app/src/main/java/com/gh4a/activities/DiffViewerActivity.java

```diff
@@ -453,9 +426,13 @@ protected void handleUrlLoad(Uri uri) {
         boolean isComment = idParam != null;
         long id = isComment ? Long.parseLong(idParam) : 0L;
 
-        CommentActionPopup p = new CommentActionPopup(id, line, lineText, leftLine, rightLine,
-                mLastTouchDown.x, mLastTouchDown.y, isComment, isRightLine);
-        p.show();
+        if (idParam == null) {
```

Comment:
Yes, I think so. Previously only the 'add comment' item applied, but now we also have 'share'.